weld/subsystem/src/main/java/org/jboss/as/weld/services/bootstrap/ProxyServicesImpl.java

```diff
@@ -90,12 +89,12 @@ private ClassLoader _getClassLoader(Class<?> proxiedBeanType) {
                 // this class comes from a static module
                 // first, check if we can use its classloader to load proxy classes
                 final Module definingModule = loader.getModule();
-                Boolean hasWeldDependencies = processedStaticModules.get(definingModule.getIdentifier());
+                Boolean hasWeldDependencies = processedStaticModules.get(definingModule.getIdentifier().toString());
```

Comment:
Please use definingModule.getName() instead of definingModule.getIdentifier().toString() - here and everywhere below

Reply:
Why?

Reply:
Thank you. This is updated.